package.json

```diff
@@ -20,7 +20,8 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
-    "array-changes": "1.0.3",
+    "array-changes": "1.2.1",
+    "array-changes-async": "2.0.2",
```

Comment:
@bruderstein and I talked briefly about consolidating the libraries. Maybe even onboarding a copy of arraydiff/arraydiff-async. The original (https://github.com/derbyjs/arraydiff) hasn't seen a commit since 2013, and they haven't yet reacted to my PR (https://github.com/derbyjs/arraydiff/pull/2).